public/docs/js/latest/glossary.jade

```diff
@@ -1,14 +1,35 @@
 include _util-fns
-+includeShared('{ts}', 'intro')
-+includeShared('{ts}', 'a2')
-+includeShared('{ts}', 'b-c')
-+includeShared('{ts}', 'd1')
-+includeShared('{ts}', 'd2')
-+includeShared('{ts}', 'e1')
-+includeShared('{ts}', 'e2')
-+includeShared('{ts}', 'f-l')
-+includeShared('{ts}', 'm1')
-+includeShared('{ts}', 'n-s')
-+includeShared('{ts}', 't1')
-+includeShared('{ts}', 't2')
-+includeShared('{ts}', 'u-z')
\ No newline at end of file
+// From ts/glossary.jade, the folder ts/latest/_fragments is generated which contains a bunch of partial files.
```

Comment:
@petebacondarwin @naomiblack I didn't want to touch `includeShared` because it is somewhat difficult to debug and it's used quite extensively in the codebase. The safer route was to include the partial files for the glossary manually, which is probably a more clear approach.